From 68cc50634d12ae5f43bcb8dafafc529c55b43872 Mon Sep 17 00:00:00 2001
From: Michal Galka <michal.galka@collabora.com>
Date: Thu, 13 May 2021 15:43:44 +0200
Subject: [PATCH 3/3] CHROMEOS Update config file paths

Configuration on chromeos.kernelci.org has been moved. Paths are updated
to reflect current file locations.

Signed-off-by: Michal Galka <michal.galka@collabora.com>
---
 app/server.py           | 2 +-
 app/taskqueue/celery.py | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/app/server.py b/app/server.py
index ac75ed80747d..92c623db946e 100755
--- a/app/server.py
+++ b/app/server.py
@@ -37,7 +37,7 @@ import utils.database.redisdb as redisdb
 import utils.db
 
 
-DEFAULT_CONFIG_FILE = "/etc/kernelci/api.chromeos.kernelci.org/" \
+DEFAULT_CONFIG_FILE = "/etc/kernelci/chromeos.kernelci.org/" \
                       "kernelci-backend.cfg"
 
 topt.define(
diff --git a/app/taskqueue/celery.py b/app/taskqueue/celery.py
index 5a4078e9a9e2..445a3812f41e 100644
--- a/app/taskqueue/celery.py
+++ b/app/taskqueue/celery.py
@@ -35,7 +35,7 @@ import taskqueue.celeryconfig as celeryconfig
 import taskqueue.serializer as serializer
 
 
-CELERY_CONFIG_FILE = "/etc/kernelci/api.chromeos.kernelci.org/" \
+CELERY_CONFIG_FILE = "/etc/kernelci/chromeos.kernelci.org/" \
                      "kernelci-celery.cfg"
 TASKS_LIST = [
     "taskqueue.tasks.bisect",
-- 
2.20.1

